coalib/bearlib/naming_conventions/__init__.py

```diff
@@ -73,6 +73,8 @@ def to_snakecase(string):
     'already_snake_case'
     >>> to_snakecase('   string  ')
     '___string__'
+    >>> to_snakecase('Hello.World')
+    'hello.world'
```

Comment:
can you change the testcase to `Hello.ThisWorld`? I think this will *not* be correctly transformed to `hello.this_world`